docker-compose/docker-compose.yml

```diff
@@ -49,6 +49,21 @@ services:
       - rabbitmq
     volumes:
       - ../:/app
+    
+  adminUI:
+    image: nmshd-admin-ui
+    container_name: admin-ui
+    hostname: admin-ui
+    build:
+      dockerfile: Dockerfile.debug
+      context: "../AdminUI/"
+    ports:
+      - "5020:4200"
+    depends_on:
+      - "adminAPI"
+    volumes:
+      - ../AdminUI/src:/app/src/:ro
+    command: "ng serve --host 0.0.0.0 --poll 1000"
```

Comment:
Why don't you specify this as the entrypoint of the Dockerfile?

Reply:
Yeah, I think I would move it to the `Dockerfile`. The whole `Dockerfile.debug` doesn't work, when you don't mount the files. So why should we make an exception for the entrypoint? :)

Reply:
The entrypoint is not part of the building itself. Makes sense